.gitlab-ci.yml

```diff
@@ -82,6 +82,11 @@ Build and push image:
       docker build \
       --build-arg "NPM_REGISTRY=https://nexus.osdc.io/repository/npm-all/" \
       --build-arg "BUILD_SHORT_SHA=$CI_COMMIT_SHORT_SHA" \
+      --build-arg "BASE_VERSION=3.0.1-rc2" \
```

Comment:
Is it because we don't have a 3.0.1 yet?

Reply:
right